pingcap/br
# web: update Lightning web interface dependencies

### What problem does this PR solve? 

The Lightning web interface dependency is outdated.

### What is changed and how it works?

`npm up` and then recompile the web interface.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
    * run `bin/tidb-lightning --server-mode --status-addr 127.0.0.1:9999` and check that the page works.

Code changes

Side effects

Related changes

### Release Note

 - No release note
